packages/expo-maps/android/src/main/java/expo/modules/maps/Records.kt

```diff
@@ -64,6 +64,20 @@ data class MarkerRecord(
   val showCallout: Boolean = true
 ) : Record
 
+data class PolylineRecord(
+  @Field
+  val coordinates: List<Coordinates> = emptyList(),
+
+  @Field
+  val geodesic: Boolean = false,
+
+  @Field
+  val color: String = "#000000",
```

Comment:
Color should be handled differently to ensure compatibility with strings and other formats.

For example, you can see how it's implemented in `expo-image`, where the color is represented as an `Int` on the native side (see https://github.com/expo/expo/blob/32f07127438a21f288aa7f3334881fb70b15064c/packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt#L226 and https://github.com/expo/expo/blob/32f07127438a21f288aa7f3334881fb70b15064c/packages/expo-image/src/ExpoImage.tsx#L80).